app/controllers/api_controller.rb

```diff
@@ -9,14 +9,7 @@ class ApiController < ActionController::API
   rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
 
   def load_user_from_request
-    @user = {}
-    if request.get? && request.params[:loggedUser].present?
-      @user = JSON.parse(request.params[:loggedUser]) || {}
-    end
-
-    if request.post? or request.patch? or request.delete? or request.put?
-      @user = request.params[:loggedUser] || {}
-    end
+    @user = get_user_from_request(request)
```

Comment:
the `load_user_from_request` calls the `get_user_from_request` method.... evil naming at work